evmrpc/txpool.go

```diff
@@ -0,0 +1,72 @@
+package evmrpc
+
+import (
+	"context"
+	"math/big"
+	"strconv"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/sei-protocol/sei-chain/x/evm/keeper"
+	rpcclient "github.com/tendermint/tendermint/rpc/client"
+)
+
+type TxPoolAPI struct {
+	tmClient     rpcclient.Client
+	keeper       *keeper.Keeper
+	ctxProvider  func(int64) sdk.Context
+	txDecoder    sdk.TxDecoder
+	txPoolConfig *TxPoolConfig
+}
+
+type TxPoolConfig struct {
+	maxNumTxs int
+}
+
+func NewTxPoolAPI(tmClient rpcclient.Client, k *keeper.Keeper, ctxProvider func(int64) sdk.Context, txDecoder sdk.TxDecoder, txPoolConfig *TxPoolConfig) *TxPoolAPI {
+	return &TxPoolAPI{tmClient: tmClient, keeper: k, ctxProvider: ctxProvider, txDecoder: txDecoder, txPoolConfig: txPoolConfig}
+}
+
+// For now, we put all unconfirmed txs in pending and none in queued
+func (t *TxPoolAPI) Content(ctx context.Context) map[string]map[string]map[string]*RPCTransaction {
+	content := map[string]map[string]map[string]*RPCTransaction{
+		"pending": make(map[string]map[string]*RPCTransaction),
+		"queued":  make(map[string]map[string]*RPCTransaction),
+	}
+
+	total := t.txPoolConfig.maxNumTxs
+	resUnconfirmedTxs, err := t.tmClient.UnconfirmedTxs(ctx, nil, &total)
+	if err != nil {
+		return nil
+	}
+
+	sdkCtx := t.ctxProvider(LatestCtxHeight)
+	signer := ethtypes.MakeSigner(
+		t.keeper.GetChainConfig(sdkCtx).EthereumConfig(t.keeper.ChainID(sdkCtx)),
+		big.NewInt(sdkCtx.BlockHeight()),
+		uint64(sdkCtx.BlockTime().Second()),
```

Comment:
`.Second()` should be `.Unix()`. Can you also fix it here https://github.com/sei-protocol/sei-chain/blob/evm/evmrpc/tx.go#L117 and here https://github.com/sei-protocol/sei-chain/blob/evm/evmrpc/tx.go#L147